indolaw-nextjs/components/TableOfContents.tsx

```diff
@@ -73,6 +96,7 @@ export default function TableOfContents(props: { law: Complex }): JSX.Element {
         <span onClick={() => setIsExpanded(false)}>
           <i className="material-icons style">chevron_left</i>Kembali
         </span>
+        {darkModeButton}
```

Comment:
<img width="590" alt="Screen Shot 2021-03-24 at 7 33 41 PM" src="https://user-images.githubusercontent.com/14287367/112410114-eb642c00-8cd7-11eb-9cbd-c3033ee03578.png">

@willemchua the button should be vertically centered like 'Daftar Isi'